src/main/java/seedu/exchangecoursemapper/command/Command.java

```diff
@@ -9,6 +9,14 @@
 import static seedu.exchangecoursemapper.constants.Commands.FILE_PATH;
 
 public abstract class Command {
+    protected static final String FILE_PATH = "/database.json";
```

Comment:
Perhaps this can be removed too! As we are using the FILE_PATH constant from our Commands class in our Commands package.